plugins/e57/libE57Format/CMakeLists.txt

```diff
@@ -30,88 +30,20 @@
 # If you find any errors or have suggestion to improve the build script:
 # patches are most welcome! Please send them to the development mailing list.
 
-
-cmake_minimum_required(VERSION 3.1.0 FATAL_ERROR)
```

Comment:
Why was this deleted? It is required if you want to build the e57 stuff stand-alone?

Reply:
This isn't for our E57 stages.  It's for the E57 support library.